server/deckUtils.go

```diff
@@ -66,6 +102,7 @@ func shuffleCards(a []model.Card) []model.Card {
 // Shuffles the deck before returning it.
 // This function is not necessarily efficient - feel free to optimize.
 func generateShuffledDeck() []model.Card {
+	// TODO - use getDeckConfigByPlayerSize() instead of getDeckConfig()
 	colors, standardCardCounts, wildCardCounts := getDeckConfig()
```

Comment:
I can switch this to use the new getDeckConfig function - just want to know what you guys think.

Reply:
What if instead of duplicating code in `getDeckConfig` and `getDeckConfigByPlayerSize`, we pass in the number of players straight to `generateShuffledDeck` and add the line `numDecks := numDecksToUse(numPlayers)` to it? Then we could just do another for loop for the number of decks so that lines 111 and 118 happen twice if there's 2 decks, thrice if there's 3, etc.

Reply:
Another way to not duplicate that code would be to call `generateShuffledDeck` for the # decks we need and shuffle them all together.

Reply:
I like this idea, but I think you could get rid of the duplication by just removing `getDeckConfig()` entirely. There's nothing that function does that `getDeckConfigByPlayerSize(1)` doesn't do.

I would pass the number of players into `generateShuffledDeck()` and replace the `getDeckConfig()` there with `getDeckConfigByPlayerSize(numPlayers)`, at the very least. generating multiple shuffled decks and then re-shuffling them together seems like more work than needed.